index.rst

```diff
@@ -15,5 +10,241 @@ Change Record
+
+Baseline Long-Term Plan
+-----------------------
+
+Planning packages, together with milestones form the *Baseline Plan*. The plan is under change control.
+
+The plan is stable. Any change to the plan, including changing scope, cost or schedule, must be approved
+by the appropriate change control body:
+
+* Change Control Board approval for any non-trivial change to a planning package or Level 1 and
+  Level 2 milestone
+
+* DM's TCT approval for Level 3 milestones
+
+* no approval is needed for Level 4 milestones, however any non-trivial change must be communicated
+  to the DM team
+
+* trivial changes, such as small corrections / clarifications to narrative that do not affect
+  scope, time or budget are allowed without approval.
+
+Typically, adjustments to Level 1, 2 and 3 milestones are made every 6 or 12 months. Changes to Level 4
+milestones can occur more frequently.
+
+Short-Term Planning
+-------------------
+
+The short-term plan is managed through *epics* and *stories*, and executed through *cycles*.
+
+In the EVMS world, epics map directly to *activities*, and stories map to *activity steps*.
+
+Epics and Stories
+~~~~~~~~~~~~~~~~~
+
+Typically, planning packages encompass relatively large blocks of work. To plan and execute work
+in details, finer-grain planning is required. This is implemented through epics and stories.
+
+Each epic captures a non-trivial work that is associated with a subset of work defined in a
+planning package. Each epic has a clear description, well defined deliverables, and
+relationships with a planning package (an epic *implements* a PP), and a milestone (an epic
+*blocks* a milestone). Epics are resource loaded, and have start and end dates defined.
+
+Each epic is broken into smaller chunks of work, called *stories*. Stories are primarily used
+to define and manage the short-term activities of individual developers.
+
+Epics and stories are used to track all software work, as well as work on delivering hardware and
+services that are tightly coupled with software (example: "Deliver OpenStack-based Test Cluster
+running RedHat 6.1".) Details of work on standalone activities do not have to be captured on daily
+bases through stories (example: "Deliver 10Gbit link between Chile and USA")
+
+.. _cycles-and-releases:
+
+Cycles and Releases
+~~~~~~~~~~~~~~~~~~~
+
+The work is executed through 6-month *cycles*. The scope of work for each cycle is driven by the
+Baseline Plan, in particular by milestones. Each cycle ends with a new software release. Releases are
+thoroughly tested and documented.
+
+Cycles are planned through *epics*. Each epic must be defined such that it can be completed
+within a single cycle.
+
+Once a plan is defined for a given cycle, it is loaded to Primavera and changed-controlled. A plan for
+a cycle is loaded to Primavera during the month proceeding the start of the cycle.
+
+Any non-trivial adjustments to the plan that affect scope, schedule or budget must be approved
+by CCB.
```

Comment:
My understanding is that after the six-month plan has been started, it can only be changed through an LCR to the LSST CCB. That's regardless of how it relates to the long term plan, which might be changed with approval from the TCT (or even without any approval at all), as above.

I'll edit the text to say this, but @jbecla should confirm.